src/structures/Items.ts

```diff
@@ -41,7 +41,7 @@ class Items extends Collection<number, Item | PartialItem> {
 
 	public get(item: ItemResolvable): Item | PartialItem | undefined {
 		const id = this.resolveID(item);
-		if (!id) return undefined;
+		if (!id && id !== 0) return undefined;
```

Comment:
This should be `if (typeof id === "undefined")` - also it seems like theres a type bug here, as TS doesnt say it can be undefined here

Reply:
Done. Tested. Works the same 👍🏽